### DISCUSSION TITLE
Ps1 games audio slowdown? Tomba 2

### DISCUSSION THREAD

**Discussion by CloudTF25** [OnionUI/Onion#502] [Q&A] [answered]
Reposted from the issues page

So when you play the first level and allow the music to change to a different track after a minute. The new track will stutter and slowdown.

ps: i'm using thepsxonpsp660.bin and a clean dump of the game in CHD format.

video below shows the issue in action

https://user-images.githubusercontent.com/67326185/173454446-414992c1-791d-45de-9ddc-e8bb37c10bd5.mp4

It was suggested that prehaps they could be a fix to this? Or this can be a suggested enhancement to onion OS? Perhaps another low cost performance PS1 emulator could fix this?

Thanks for reading. I appreciate it

**Comment by schmurtzm** [comment, marked as answer]
This has been solved thank to your issue. The updated PCSX is now included in the current Onion 4.x versions.
